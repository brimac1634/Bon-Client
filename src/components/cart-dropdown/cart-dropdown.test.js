import { shallow } from "enzyme";
import React from "react";
import CartDropdown from "./cart-dropdown.component";

it("expect to render CartDropdown component", () => {
  const mockItems = [
    {
      name: "test item",
      quantity: 5,
      price: 60,
      images: ["test_image"]
    }
  ];
  expect(
    shallow(<CartDropdown cartItems={mockItems} />).debug()
  ).toMatchSnapshot();
});
