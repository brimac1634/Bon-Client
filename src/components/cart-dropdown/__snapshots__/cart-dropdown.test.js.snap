// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render CartDropdown component 1`] = `
"<ContextConsumer>
  [function]
</ContextConsumer>"
`;
